syntax = "proto3";
import "google/protobuf/empty.proto";
package data;

message Data {
  string pollution_data = 1;
  string meteo_data = 2;
  int32 timestamp = 3;
}

service DataRPC {
  rpc GetData (Data) returns (google.protobuf.Empty) {}
}
